public/app/stores/PermissionsStore/PermissionsStore.ts

```diff
@@ -50,9 +51,10 @@ export const NewPermissionsItem = types
     },
   }))
   .actions(self => ({
-    setUser(userId: number, userLogin: string) {
+    setUser(userId: number, userLogin: string, userAvatarUrl: string) {
```

Comment:
Same is needed for setTeam. When you add a new team permission the team avatar is not rendered